MIGRATION.md

```diff
@@ -130,7 +187,7 @@ The deprecated `text[Kilo|Mega|Giga]` style mixins were replaced by a single `ty
 
 #### Heading `as` props
 
-The `as` prop is now required in both the **Headline** and the **SubHeadline** components, and restricted to HTML heading elements. This makes heading structure explicit, and ensures that heading components render semantic heading elements.
+The `as` prop is now required in both the **Headline** and the **SubHeadline** components. Intentionally setting the heading level ensures a consistent and accessible page structure. The `as` prop values was also restricted to HTML heading elements to ensures that heading components render semantic heading elements.
```

Comment:
```suggestion
The `as` prop is now required in both the **Headline** and the **SubHeadline** components. Intentionally setting the heading level ensures a consistent and accessible page structure. The `as` prop values were also restricted to HTML heading elements to ensure that heading components render semantic heading elements.
```

Reply:
Oh my, I think I was getting a bit tired by this point 🙈